src/VoyagerHooksServiceProvider.php

```diff
@@ -118,10 +119,14 @@ public function addHookMenuItem()
 
     public function addHookPermissions()
     {
-        Permission::firstOrCreate([
-            'key'        => 'browse_hooks',
-            'table_name' => null,
-        ]);
+        $permission = Permission::firstOrCreate([
+                        'key'        => 'browse_hooks',
+                        'table_name' => null,
+                    ]);
+
+        $role = Role::where('name', 'admin')->firstOrFail();
```

Comment:
We really can't assume the role will be named 'admin'

Reply:
You have a point. The thing is that voyager automatically creates an Admin role, and adds all permissions to it. So unless you manually change the highest role to something else, it does not matter.

In the config file from voyager on line 175, they have a setting which changes the default role to add all the permissions to. Maybe that's an idea. (The setting is called `default_role`)

If you have any suggestions let me know, I'll be happy to implement them.